JASP-Engine/JASP/R/ttestonesample.R

```diff
@@ -487,8 +482,19 @@ TTestOneSample <- function(dataset = NULL, options, perform = "run",
 	for (i in .indices(options$variables)) {
 
 		var <- options$variables[[i]]
-
 		descriptivesPlot <- list("title" = var)
+		#### new error handling
+		errors <- .hasErrors(dataset, perform, message = 'short', type = c('observations', 'variance', 'infinity'),
```

Comment:
From what I can see the plot below this would still get made even when errors are found.
Is that intended?